cmd/analyzer/main.go

```diff
@@ -84,12 +78,27 @@ func _main(cmdlineArgs []string) error {
 	if err != nil {
 		return fmt.Errorf("error generating cloud config from input vpc resources file: %w", err)
 	}
+	outputPerVPC := make([]*vpcmodel.VPCAnalysisOutput, len(vpcConfigs))
+	i := 0
+	var vpcAnalysisOutput *vpcmodel.VPCAnalysisOutput
 	for _, vpcConfig := range vpcConfigs {
-		err = analysisPerVPCConfig(vpcConfig, inArgs)
+		vpcAnalysisOutput, err = analysisPerVPCConfig(vpcConfig, inArgs)
```

Comment:
```suggestion
		vpcAnalysisOutput, err := analysisPerVPCConfig(vpcConfig, inArgs)
```
No need for the var defined outside of the loop

Reply:
I originally wrote as you suggested, but then I got a linter issue ` shadow: declaration of "err" shadows declaration at line ...`

Reply:
you can simply rename `err` to `err2` on such cases

Reply:
ok, changed